charts/alfresco-common/templates/_helpers-activemq.tpl

```diff
@@ -1,3 +1,27 @@
+{{/*
+Validate ActiveMQ has a failover transport URL
+
+Usage: include "alfresco-common.activemq.url.valid" "URL"
+
+*/}}
+{{- define "alfresco-common.activemq.url.valid" -}}
```

Comment:
try to have a name which reflect the current behaviour - remember that changing the implementation without changing the template name is risky

something like `alfresco-common.activemq.url.withFailover`?